scripts/convert_publish.py

```diff
@@ -38,6 +38,7 @@ def __init__(self, golos_db, publish_db, cyberway_db, cache_period = 10000):
 
     def fill_exists_accs(self):
         cw_accounts = self.cyberway_db['account']
+        cw_accounts = cw_accounts.with_options(codec_options = bson.CodecOptions(unicode_decode_error_handler="ignore"))
```

Comment:
It's better to use projection for get only `name` field from database.
For this use second argument in method `find` in next line. So:
```
for doc in cw_accounts.find({},{'name':1})
```